pytest_pylint/tests/test_pytest_pylint.py

```diff
@@ -28,7 +28,7 @@ def test_nodeid(testdir):
     """Verify our nodeid adds a suffix"""
     testdir.makepyfile('import sys')
     result = testdir.runpytest('--pylint', '--collectonly')
-    assert '::PYLINT' in result.stdout.str()
+    assert 'PYLINT' in result.stdout.str()
```

Comment:
Same here, I'd prefer to keep the ::PYLINT, just remove the duplicated full path issue